### DISCUSSION TITLE
Badge count

### DISCUSSION THREAD

**Discussion by nvmkpk** [spacecowboy/Feeder#207] [Ideas]
If I add a tag to a feed, the tag shows as a folder that shows the total unread count. If I have one unread article in the feed, the app shows 1 for the feed, 1 for the tag and 1 for 'All' but the launcher shows the count as 2 in the badge.

The count in the badge should only show the total from the feeds.

**Comment by spacecowboy** [comment]
> but the launcher shows the count as 2 in the badge.

Feeder does not support any kind of badge in the launcher.

So... I'm not quite sure what the number represents or why it is there. What launcher/device is this?

**Reply by nvmkpk** [reply]
I am using Microsoft Launcher in Pixel 7 Pro.

**Reply by spacecowboy** [reply]
A quick google suggests the badges in Microsoft Launcher is a count of notifications.

Either way, if this isn't working, it's a problem with Microsoft Launcher.

**Reply by nvmkpk** [reply]
I don't know what is going on but this is the only app I see this issue with. All other apps show correct count.
